### DISCUSSION TITLE
The focus moves when an item is selected

### DISCUSSION THREAD

**Discussion by Alexandring** [ArthurSonzogni/FTXUI#452] [Q&A]
Hello. I am faced with the following problem.
Select an item in the menu with the mouse and the selected item moves to the middle of the window. The same thing happens when I use containers. How do I disable this feature? I want the selected element to remain in its place and scrolling does not happen.

**Comment by ArthurSonzogni** [comment]
I moved this discussion into:
https://github.com/ArthurSonzogni/FTXUI/issues/453
